### DISCUSSION TITLE
Wayland?

### DISCUSSION THREAD

**Discussion by 9acca9** [BuddiesOfBudgie/budgie-desktop#641] [Q&A] [answered]
Hi budgie desktop support Wayland?
thanks

**Comment by EbonJaeger** [comment, marked as answer]
Not yet. Wayland support is something that we are all actively working on, though.

As it stands now, the `main` git branch of budgie-desktop only runs on Wayland, even though there are still some areas that need work. It also requires work being done in other projects like `budgie-session`, Magpie, and more. It is currently not ready to be used by those outside the Budgie team.

**Reply by eriad25** [reply]
Based on your response is it safe to assume that 10.10 should have full wayland support? I mean you guys said on blog that budgie 11 will be wayland only.

**Reply by fossfreedom** [reply]
10.10 will only have wayland support.  X11 will not be supported with 10.10.  More details in our blog https://buddiesofbudgie.org/blog/state-of-the-budgie-2024

**Reply by eriad25** [reply]
I saw it on blog. I didn't read it to the fullest at first but i saw it. Still thanks for info.

**Reply by 9acca9** [reply]
Oh, thanks!
Amazing!
